aiounifi/devices.py

```diff
@@ -345,3 +384,77 @@ def __getitem__(self, obj_id: int) -> Port:
     def __iter__(self) -> Iterator[Union[int, str]]:
         """Iterate over ports."""
         return iter(self.ports)
+
+
+class Outlet:
+    """Represents an outlet."""
+
+    def __init__(self, raw: dict) -> None:
+        """Initialize outlet."""
+        self.raw = raw
+
+    @property
+    def name(self) -> str:
+        """Name of outlet"""
+        return self.raw.get("name", "")
+
+    @property
+    def index(self) -> int:
+        """Outlet index."""
+        return self.raw.get("index")
```

Comment:
Index always exist? Current typing suggests that an int will always be returned, but here None can also be returned.

I think we can expect it to exist.

If you actually expect there be situations where index will be None please add a test for it.

Reply:
I agree that index will likely always exist.  I'm a bit out of my depth here.  I see that None will be returned if self.raw.get("index") doesn't produce a value, but I'm not sure what to do about that.  I don't think it makes sense to pick a default integer to return instead.  The Port class uses Optional[int] for port_idx which then allows None to be returned.  Should I set this up that way?

Reply:
Ah, I see from the example about relay_state below.  I'll remove the .get() part.